File: app/src/main/java/com/cb3g/channel19/Coordinates.java
package com.cb3g.channel19;

public class Coordinates {
    String userId;
    String handle;
    String profile;
    double latitude;
    double longitude;
    float bearing;
    float speed;
    double altitude;

    public Coordinates() {
    }

    public Coordinates(String userId, String handle, String profile, double latitude, double longitude, float bearing, float speed, double altitude) {
        this.userId = userId;
        this.handle = handle;
        this.profile = profile;
        this.latitude = latitude;
        this.longitude = longitude;
        this.bearing = bearing;
        this.speed = speed;
        this.altitude = altitude;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getHandle() {
        return handle;
    }

    public void setHandle(String handle) {
        this.handle = handle;
    }

    public String getProfile() {
        return profile;
    }

    public void setProfile(String profile) {
        this.profile = profile;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    public float getBearing() {
        return bearing;
    }

    public void setBearing(float bearing) {
        this.bearing = bearing;
    }

    public float getSpeed() {
        return speed;
    }

    public void setSpeed(float speed) {
        this.speed = speed;
    }

    public double getAltitude() {
        return altitude;
    }

    public void setAltitude(double altitude) {
        this.altitude = altitude;
    }
}

File: app/src/main/java/com/cb3g/channel19/Coordinates.kt
package com.cb3g.channel19

class Coordinates {
    var userId = ""
    var handle = ""
    var profile = ""
    var latitude = 0.0
    var longitude = 0.0
    var bearing = 0f
    var speed = 0f
    var alititude = 0.0

    constructor()

    constructor(userId: String, handle: String, profile: String, latitude: Double, longitude: Double, bearing: Float, speed: Float, altitude: Double) {
        this.userId = userId
        this.handle = handle
        this.profile = profile
        this.latitude = latitude
        this.longitude = longitude
        this.bearing = bearing
        this.speed = speed
        this.alititude = altitude
    }


}
